tests/bluetooth/mesh_shell/prj.conf

```diff
@@ -16,10 +16,12 @@ CONFIG_FLASH_MAP=y
 CONFIG_NVS=y
 CONFIG_SETTINGS=y
 
+CONFIG_BT_MESH_USES_MBEDTLS_PSA=y
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=1024
```

Comment:
Right, this can only be done in the option itself.

But, actually, I can't seem to find where malloc is used in mbedtls. Why do we need this option?

Reply:
not necessary anymore after configuration via `config-tls-generic.h`

Reply:
Yes, the current solution with keys is temporary. After normal implementation the keys should be imported once when Mesh stack gets them. However, it is not possible to refuse fully not persistent key usage i.e. during Diffie-Hellman shared secret generation.